### PR TITLE
Switch some more config values to XPath syntax

### PR DESCRIPTION
**Pull request by KarlStraussberger**
This reduces variations in property handling.

There are still some left, but to me it looks a little bit clearer...

### CODE REVIEW

File: src/config/config_generator.cc [hunk 1]
```diff
@@ -374,42 +359,42 @@ void ConfigGenerator::generateOnlineContent()
...
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_FIRST, YES);
+    setValue(vlcmpeg, ATTR_TRANSCODING_PROFILES_PROFLE_ACCOGG, YES);
 
-    agent = setValue("/transcoding/profiles/profile/agent/", "", true);
+    agent = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/agent/", "", true);
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_COMMAND, "vlc");
     setValue(agent, ATTR_TRANSCODING_PROFILES_PROFLE_AGENT_ARGS, "-I dummy %in --sout #transcode{venc=ffmpeg,vcodec=mp2v,vb=4096,fps=25,aenc=ffmpeg,acodec=mpga,ab=192,samplerate=44100,channels=2}:standard{access=file,mux=ps,dst=%out} vlc:quit");
 
-    buffer = setValue("/transcoding/profiles/profile/buffer/", "", true);
+    buffer = setValue(std::string(ConfigManager::mapConfigOption(ATTR_TRANSCODING_PROFILES_PROFLE)) + "/buffer/", "", true);
```
Comment (whyman) [inline src/config/config_generator.cc:397]: maybe a fmt::format would be nicer here?

File: web/js/jquery.gerbera.config.js [hunk 2]
```diff
@@ -488,7 +488,9 @@ $.widget('grb.config', {
               // already handled
             break;
             default:
-              console.log(`${item.item}: Unknown type '${item.type}'`);
+              if (('aid' in item)) {
```
Comment (whyman) [inline web/js/jquery.gerbera.config.js:491]: I assume this is not in here intentionally

Comment (KarlStraussberger) [reply web/js/jquery.gerbera.config.js]: It is. Added comment to code to explain. And also added entries to json, so properties have help.

File: src/config/config_generator.cc (hunk 1 above)
Comment (KarlStraussberger) [reply src/config/config_generator.cc]: Leftovers... Cleaned them out.

**Pull request merged**